soc/arc/snps_nsim/Kconfig.defconfig.sem

```diff
@@ -16,7 +16,7 @@ config NUM_IRQS
 	default 30
 
 config ARC_MPU_VER
-	default 4
+	default 8
```

Comment:
As we've dropped nsim props / mdb args changes we also need to drop this change.